src/phoenix/experimental/evals/evaluators.py

```diff
@@ -0,0 +1,172 @@
+import json
+from dataclasses import dataclass
+from typing import Any, List, Mapping, Optional, Protocol, cast
+
+from .models.openai import OpenAIModel, OpenAIResponse
+from .templates import PromptTemplate
+
+Record = Mapping[str, Any]
+
+
+@dataclass
+class Eval:
```

Comment:
```suggestion
class EvalResult:
```
This is a bit confusing without a proper name